integration-tests/java/build.gradle

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 Infostellar, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java'
+
+sourceCompatibility = '1.8'
+targetCompatibility = '1.8'
+
+dependencies {
+    implementation project(':api')
+
+    implementation 'com.google.auth:google-auth-library-oauth2-http:0.16.1'
+    implementation 'com.google.dagger:dagger:2.23.1'
+    ['auth', 'netty', 'protobuf', 'stub'].each {
+        implementation "io.grpc:grpc-$it:1.21.0"
+    }
+    implementation 'com.typesafe:config:1.3.4'
+    implementation 'io.netty:netty-tcnative-boringssl-static:2.0.25.Final'
+    implementation 'org.slf4j:slf4j-simple:1.7.26'
+
+    annotationProcessor 'com.google.dagger:dagger-compiler'
```

Comment:
I think the version is needed here too:

com.google.dagger:dagger-compiler:2.23.1
and same on the line below.

I think it works for us because of a side effect of the curiostack plugin setting things up for us.